test/wrapper_test.rb

```diff
@@ -3224,6 +3224,40 @@ def test_cannot_simplify_service_setup_duration
     }
   end
 
+  def test_simplify_complex_multi_pickup_or_delivery_shipments
+    # check service count after simplification
+    vrp = TestHelper.load_vrp(self, fixture_file: 'vrp_multipickup_singledelivery_shipments')
+    service_count = vrp.services.size
+    OptimizerWrapper.config[:services][:demo].simplify_complex_multi_pickup_or_delivery_shipments(vrp)
+    assert_equal service_count + 7, vrp.services.size, 'simplify should have created 7 new services'
+
+    # Solve WITHOUT simplification
+    simplifier_called = false
+    result_wo_simplify = Wrappers::Wrapper.stub_any_instance(:simplify_complex_multi_pickup_or_delivery_shipments,
+                                                                    proc{
+                                                                      simplifier_called = true
+                                                                      nil
+                                                                    }) do
```

Comment:
it does look weird, have no idea why rubocop doesn't complain.

Reply:
Ah, found it, it is because of github parsing, line 3236 is cut in half when you put that line back together   `proc{` is aligned with the other argument of `stub_any_instance` -- i.e. it basically is

Reply:
But shouldn't ```proc``` be aligned with ```:simplify_complex_multi_pickup_or_delivery_shipments``` (or with anything else) ?